influxql/parser.go

```diff
@@ -1473,13 +1473,6 @@ func (p *Parser) parseDropContinuousQueryStatement() (*DropContinuousQueryStatem
 func (p *Parser) parseFields() (Fields, error) {
 	var fields Fields
 
-	// Check for "*" (i.e., "all fields")
-	if tok, _, _ := p.scanIgnoreWhitespace(); tok == MUL {
```

Comment:
This previously assumed a single wildcard, and they returned.  so something like `select *, foo` would fail, but `select foo, *` would pass.  In general, it's code bloat now and the code below handles a wildcard fine.